apps/prairielearn/src/migrations/20250924190742_course_instance_publishing_extensions__add.sql

```diff
@@ -0,0 +1,21 @@
+CREATE TABLE course_instance_publishing_extensions (
+  id BIGSERIAL PRIMARY KEY,
+  course_instance_id BIGINT NOT NULL REFERENCES course_instances ON DELETE CASCADE ON UPDATE CASCADE,
+  name TEXT,
+  end_date TIMESTAMP WITH TIME ZONE NOT NULL
+  -- These records will be hard deleted.
```

Comment:
Is this comment needed? That should be clear from the lack of a `deleted_at` column.